.github/workflows/release_mac.yml

```diff
@@ -43,7 +41,21 @@ jobs:
       uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5.1.0
       with:
         python-version: ${{ matrix.python-version }}
-        architecture: ${{ matrix.host-architecture }}
+
+    - name: override python 3.8 install
+      if: matrix.python-version == '3.8'
+      run: |
+        # the universal2 installer used by setup-python can't target macos 10.15 (minimum target is 11.0)
```

Comment:
Looks like Apple has ended support for Catalina on November 30, 2022. We should be good targeting 11.0 if that simplifies the process

Reply:
If targeting 11.0 is ok then yes, this whole block can be dropped.